src/newsGetter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+The newsGetter class is reponsible for getting the news article from the RSS feed 
+and checking for duplicates.
+*/
+
+include("article.php");
+
+class newsGetter
+{
+
+    public $lastlink;
+    
+    public function __construct()
+    {
+
+        $this->lastlink = "";
```

Comment:
Where going from an array with duplicate links to one string? This probably causes the duplicates you where talking about [here](https://github.com/Artemixed/Discord-uWuFication/pull/3#issuecomment-1039471770). Moving back to an array would fix that issue again. 

Would recommend limiting this array to 10 links (maybe more are needed) or something like that. Otherwise this would fill up your memory when running it for a while.

Reply:
Fixed in accd0ab3fd15c2b0f0ceed394b93692b9b6a3882 !